discovery-provider/src/api/v1/utils/extend_notification.py

```diff
@@ -484,7 +486,9 @@ def extend_usdc_purchase_buyer(action: NotificationAction):
             "buyer_user_id": encode_int_id(data["buyer_user_id"]),
             "seller_user_id": encode_int_id(data["seller_user_id"]),
             "amount": str(data["amount"]),
-            "extra_amount": str(data["extra_amount"]),
+            "extra_amount": str(data["extra_amount"])
```

Comment:
this isn't actually something we'd see on prod, but figured i should clean it up because this was exploding logs.